README.md

```diff
@@ -193,6 +193,115 @@ spec:
      iptables-min-sync-period: ["30s"]
 ```
 
+## Configuring Additional Networks
+Users can configure additional networks, based on [Kubernetes Network Plumbing Working Group's Kubernetes Network Custom Resource Definition De-facto Standard Version 1](https://github.com/K8sNetworkPlumbingWG/multi-net-spec/blob/master/%5Bv1%5D%20Kubernetes%20Network%20Custom%20Resource%20Definition%20De-facto%20Standard.md).
+
+* `name`: name of network attachment definition, required
+* `namespace`: namespace for the network attachment definition. The default is default namespace
+* `type`: specify network attachment definition type, required
+
+Currently, the understood values for type are:
+* `Raw`
+* `SimpleMacvlan`
+
+```yaml
+spec:
+  additionalNetworks:
+  - name: test-network-1
+    namespace: namespace-test-1
+    type: ...
+```
+
+### Configuring Raw CNI
+Users can configure network attachment definition with CNI json as following options required:
+
+* `rawCNIConfig`: CNI JSON configuration for the network attachment
+
+```yaml
+spec:
+  additionalNetworks:
+  - name: test-network-1
+    namespace: namespace-test-1
+    rawCNIConfig: '{ "cniVersion": "0.3.1", "type": "macvlan", "master": "eth1", "mode": "bridge", "ipam": { "type": "dhcp" } }'
+    type: Raw
+```
+
+This config will generate the following network attachment definition:
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  # (snip)
+  name: test-network-1
+  namespace: namespace-test-1
+  ownerReferences:
+  - apiVersion: operator.openshift.io/v1
+    # (snip)
+spec:
+  config: '{ "cniVersion": "0.3.1", "type": "macvlan", "master": "eth1", "mode": "bridge", "ipam": { "type": "dhcp" } }'
+```
+
+### Configuring SimpleMacvlan
+Users can configure macvlan network attachment definition with following parameters, all of which are optional:
+
+* `master`: master is the host interface to create the macvlan interface from. If not specified, it will be default route interface
+* `mode`: mode is the macvlan mode: bridge, private, vepa, passthru. The default is bridge
+* `mtu`: mtu is the mtu to use for the macvlan interface. if unset, host's kernel will select the value
+* `ipamConfig`: IPAM (IP Address Management) configration: dhcp or static. The default is dhcp
+
+```yaml
+spec:
+  additionalNetworks:
+  - name: test-network-2
+    type: SimpleMacvlan
+    simpleMacvlanConfig:
+      master: eth0
+      mode: bridge
+      mtu: 1515
+      ipamConfig:
+        type: dhcp
+```
+
+#### Configuring Static IPAM
+Users can configure static IPAM with following parameters:
```

Comment:
I would also mention that the IP address can come from an annotation on the pod, and how to configure that.

Reply:
Added.